tests/test-notebook/src/actions.spec.ts

```diff
@@ -912,79 +912,79 @@ describe('@jupyterlab/notebook', () => {
     describe('#extendSelectionAbove()', () => {
       it('should extend the selection to the cell above', () => {
         widget.activeCellIndex = 1;
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
         expect(widget.isSelected(widget.widgets[0])).to.equal(true);
       });
 
       it('should be a no-op if there is no model', () => {
         widget.model = null;
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
         expect(widget.activeCellIndex).to.equal(-1);
       });
 
       it('should change to command mode if there is a selection', () => {
         widget.mode = 'edit';
         widget.activeCellIndex = 1;
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
         expect(widget.mode).to.equal('command');
       });
 
       it('should not wrap around to the bottom', () => {
         widget.mode = 'edit';
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
         expect(widget.activeCellIndex).to.equal(0);
         const last = widget.widgets[widget.widgets.length - 1];
         expect(widget.isSelected(last)).to.equal(false);
         expect(widget.mode).to.equal('edit');
       });
 
       it('should deselect the current cell if the cell above is selected', () => {
-        NotebookActions.extendSelectionBelow(widget);
-        NotebookActions.extendSelectionBelow(widget);
+        NotebookActions.extendSelectionBelow(widget, false);
+        NotebookActions.extendSelectionBelow(widget, false);
         const cell = widget.activeCell;
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
         expect(widget.isSelected(cell)).to.equal(false);
       });
 
       it('should select only the first cell if we move from the second to first', () => {
-        NotebookActions.extendSelectionBelow(widget);
+        NotebookActions.extendSelectionBelow(widget, false);
         const cell = widget.activeCell;
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
         expect(widget.isSelected(cell)).to.equal(false);
         expect(widget.activeCellIndex).to.equal(0);
       });
 
       it('should activate the cell', () => {
         widget.activeCellIndex = 1;
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
         expect(widget.activeCellIndex).to.equal(0);
       });
     });
 
     describe('#extendSelectionBelow()', () => {
       it('should extend the selection to the cell below', () => {
-        NotebookActions.extendSelectionBelow(widget);
+        NotebookActions.extendSelectionBelow(widget, false);
```

Comment:
Can you add a test for the second parameter being true here and for extendSelectionAbove?

Reply:
Sure.